okta/resource_okta_group_memberships.go

```diff
@@ -123,9 +130,9 @@ func checkIfGroupHasUsers(ctx context.Context, client *okta.Client, groupId stri
 	if err := suppressErrorOn404(resp, err); err != nil {
 		return false, fmt.Errorf("unable to return membership for group (%s) from API", groupId)
 	}
-	if len(groupUsers) == 0 {
-		return false, nil
-	}
+	// if len(groupUsers) == 0 {
+	// 	return false, nil
+	// }
```

Comment:
On the commit history I see Bogdan added this as a panic bumper in #795 but I also see how that changed the behavior to #1072 . When I look at the [checkIfGroupHasUsers](https://github.com/okta/terraform-provider-okta/blob/18210979890cbc9899e711ef29ea7071a453a3f5/okta/resource_okta_group_memberships.go#L128-L167) I don't see a codepath where a panic could happen so I think this is OK.

However, just delete the lines.

Reply:
@exitcode0 I'll approve this PR after that commented code is clipped out. Thanks!